<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imageRegistration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Registration v0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loaded_Files">
     <widget class="QPushButton" name="nextToMoving">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>400</y>
        <width>98</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadNew_Button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Load New DICOM</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>551</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>551</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a Series for fixed Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="nextToComp">
      <property name="geometry">
       <rect>
        <x>467</x>
        <y>400</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Start Computing</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetMoving">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>551</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>551</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a Series for moving Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonAddMoving">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add another Moving Series</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Computing">
     <widget class="QPushButton" name="startComputing_Button">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>410</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Registration</string>
      </property>
     </widget>
     <widget class="QVTKWidget" name="vtkRenderer1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>311</width>
        <height>291</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QVTKWidget" name="vtkRenderer2" native="true">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>311</width>
        <height>291</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="hSliderFixed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>311</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hSliderMoving">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>311</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="showFit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>98</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Fit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fixed Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Moving Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>19</width>
        <height>291</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QVTKWidget" name="vtkRendererResult" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="hSliderResult">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>601</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>581</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Difference between fixed and moving image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>27</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuLoad_Files">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Files"/>
    <addaction name="actionSave_Files"/>
    <addaction name="actionShow"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuLoad_Files"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad_Files">
   <property name="text">
    <string>Load Files</string>
   </property>
  </action>
  <action name="actionSave_Files">
   <property name="text">
    <string>Save Files</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show Files</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
